contracts/test-integration/BlsSignerContractInteraction.test.ts

```diff
@@ -217,17 +217,17 @@ describe("Signer contract interaction tests", function () {
     });
 
     // TODO: Investigate why safeMint() fails with a BLS wallet address. Note - it passes in isolation
-    it("safeMint() call fails with BLS wallet address", async function () {
+    it.skip("safeMint() call fails with BLS wallet address", async function () {
```

Comment:
Do we still want this skipped?

Reply:
It seems to be passing in isolation but then failing when run with other tests. Thought it would be better to skip it in case it causes issues like some other integration tests have, then implement a fix for it. I can add a follow up issue to look into it so it's tracked.

Let me know if you have any additional thoughts

Reply:
Follow up #484

Reply:
Nope, that sounds good!  I just wanted to make sure it wasn't an accidental add.